__tests__/summary.test.ts

```diff
@@ -119,8 +119,25 @@ test('adds deprecation warning for deny-licenses option', () => {
   )
   const text = core.summary.stringify()
 
+  expect(text).not.toContain(
+    '⚠️ <strong>Deprecation Warning</strong>: The <em>deny-licenses</em> option is deprecated and will be removed in the next major version release, use <em>allow-licenses</em> instead.'
```

Comment:
nit: For a `not.toContain` test, I'd check for something short like `deny-licenses` or `Deprecation Warning` because any little text change would make this test always pass and not tell us anything useful.

Reply:
Actually, looks like this is already the case… the deprecation warning _could_ be there, but the text of the warning is different from what we're testing for.